src/character.cpp

```diff
@@ -2306,3 +2282,63 @@ std::string Character::extended_description() const
 
     return replace_colors( ss.str() );
 }
+
+template <typename Getter>
+float Character::calc_mutation_value( Getter getter ) const
```

Comment:
Why does this acept a templated getter as opposed to say, a identifier indicating the stat to use?  It seems like there are much simpler solutions, especially if it's being passed through multiple methods.